packages/mattermost-redux/src/actions/teams.ts

```diff
@@ -667,7 +672,7 @@ export function joinTeam(inviteId: string, teamId: string): ActionFunc {
             return {error};
         }
 
-        getMyTeamUnreads()(dispatch, getState);
+        getMyTeamUnreads(isCollapsedThreadsEnabled(state))(dispatch, getState);
```

Comment:
nitpick, not a blocking comment: since we are using `redux-thunk` just dispatching the action feels more "natural".
```suggestion
        dispatch(getMyTeamUnreads(isCollapsedThreadsEnabled(state)));
```